client/src/includes/initDismissibles.ts

```diff
@@ -1,65 +0,0 @@
-import { WAGTAIL_CONFIG } from '../config/wagtailConfig';
-
-/**
- * Updates the server, using a PATCH request when the toggle is clicked on a dismissible
- * element initialised by initDismissibles
- *
- * @param data - The dismissible represented as an object with keys as
- * the id and its new state: whether it is dismissed (boolean)
- *
- * @return {Promise<Response>}
- *
- * @example
- * const data = { 'dismissible-1': true, 'dismissible-2': false };
- * const wagtailConfig = {}
- *
- * updateDismissibles(data, wagtailConfig);
- */
-export function updateDismissibles(
-  data: Record<string, boolean>,
-): Promise<Response> {
-  return fetch(WAGTAIL_CONFIG.ADMIN_URLS?.DISMISSIBLES, {
-    method: 'PATCH',
-    headers: {
-      [WAGTAIL_CONFIG.CSRF_HEADER_NAME]: WAGTAIL_CONFIG.CSRF_TOKEN,
-      'Content-Type': 'application/json',
-    },
-    body: JSON.stringify(data),
-    mode: 'same-origin',
-  });
-}
-
-/**
- * Initialise dismissibles fetched from server and add click event listeners to them.
- * @return {void}
- */
-export function initDismissibles(): void {
-  // A dismissible element is marked by the data-wagtail-dismissible-id attribute.
-  const dismissibles = document.querySelectorAll<HTMLElement>(
-    '[data-wagtail-dismissible-id]',
-  );
-
-  // Initialise such elements with a default handler that performs the dismissal.
-  // This only initialises elements that are rendered by the server, so elements
-  // that are rendered by the client (e.g. React) needs to be handled separately.
```

Comment:
Maybe move some of this description to the new controller class, it gives helpful context.